apps/camera_pipe/camera_pipe_generator.cpp

```diff
@@ -459,11 +459,22 @@ void CameraPipe::generate() {
 
         Var xi, yi, xii, xio;
 
-        /* 1391us on a gtx 980. */
+        /* These tile factors obtain 1391us on a gtx 980. */
+        int tile_x = 28; 
+        int tile_y = 12;
+
+        if (get_target().has_feature(Target::D3D12Compute)) {
+            // D3D12 can only utilize a limited amount of
```

Comment:
nitpick: it's not so much d3d12, but the minimum shader model version (5.1) that we're adhering to that has this limitation.

Reply:
Changed the comment.